src/components/Breadcrumbs/Breadcrumb/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface BreadcrumbType {
```

Comment:
You can actually export types from `vue` files even while using `script setup` by exporting them from a normal `script lang="ts"` block. Consider this for other places as well